app/javascript/channels/scrape_status_channel.js

```diff
@@ -0,0 +1,15 @@
+import consumer from "./consumer"
```

Comment:
Is this ActionCable channel used yet? Is there something coming in the future?

Reply:
Good catch, this was supposed to be removed (it was a dead in I was pursuing)